mods/sample-nginx.config

```diff
@@ -82,27 +82,31 @@ server {
 
   # rewrite to front controller as default rule
   location / {
-    if ($is_args != "") {
-        rewrite ^/(.*) /index.php?pagename=$uri&$args last;
-    }
-    rewrite ^/(.*) /index.php?pagename=$uri last;
+     if (!-e $request_filename) {
+     rewrite ^(.*)$ /index.php?q=$1;
+      }
   }
+  
 
   # make sure webfinger and other well known services aren't blocked
   # by denying dot files and rewrite request to the front controller
   location ^~ /.well-known/ {
     allow all;
-    rewrite ^/(.*) /index.php?pagename=$uri&$args last;
+   if (!-e $request_filename) {
+      rewrite ^(.*)$ /index.php?q=$1;
+    }
   }
 
   # statically serve these file types when possible
   # otherwise fall back to front controller
   # allow browser to cache them
   # added .htm for advanced source code editor library
-  location ~* \.(jpg|jpeg|gif|png|ico|css|js|htm|html|ttf|woff|svg)$ {
-    expires 30d;
-    try_files $uri /index.php?pagename=$uri&$args;
-  }
+  # location ~* \.(jpg|jpeg|gif|png|ico|css|js|htm|html|ttf|woff|svg)$ {
```

Comment:
If this doesn't work, should we better drop it instead of keeping dead lines of code around?